src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap

```diff
@@ -34,7 +34,7 @@ Object {
   },
   "meta": Object {
     "analytics": Object {
-      "eventName": "Registered",
+      "eventName": undefined,
```

Comment:
Tibet, did an update to analytics get pushed in your edits?

Reply:
hmm, i will investigate

Reply:
oh i think you just needed to `yarn` first